Primera adición

castjuan opened the pull request:
Interfaces
Algunos DTs vacíos por dependencias
Clase conversación
EL GRAN CAMBIO DE "Fabrica" a "FACTORY"

castjuan committed:
Primera adición

Interfaces
Algunos DTs vacíos por dependencias
Clase conversación
EL GRAN CAMBIO DE "Fabrica" a "FACTORY"

MacielFede reviewed (approved):
Que linda aplastada de teclado hizo con sus nalgas señor

michaelramos95 reviewed (approved):
Leave a comment

MacielFede reviewed (changes requested):
Juan te falto en Conversacion.h agregar un map de Mensajes, ademas getDataConversacion tendria que ser virtual pura.

castjuan committed:
Actualizaciones

- Map de mensajes
- Clase de Mensajes vacía
- Un cambio q me había quedado de antes q no se paso

castjuan committed:
Merge branch 'main' into juan
